Base/Logic/Resources/ColorFiles/Terminology/GenericAnatomyColors-SNOMED.csv

```diff
@@ -0,0 +1,291 @@
+# Integer Label,Text Label,Segmented Property Category -- CID 7150++,Segmented Property Type,Segmented Property Type Modifier,Color
+# Version 2
+SlicerLUT=GenericAnatomyColors
+1,tissue,(T-D0050;SRT;Tissue),(T-D0050;SRT;Tissue),,rgb(128;174;128)
```

Comment:
An idea: may be you could do a first commit adding the functionality to manage terminology, and then a second one adding the data file where you would explicitly credit them ?